go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/google/go-cmp v0.5.9
 	github.com/google/uuid v1.1.2
 	github.com/hashicorp/go-azure-helpers v0.55.0
-	github.com/hashicorp/go-azure-sdk v0.20230317.1100159
+	github.com/hashicorp/go-azure-sdk v0.20230320.1165256
```

Comment:
mind reverting/resolving the merge conflicts on this one?

Reply:
Fixed.